aws/resource_aws_vpc_endpoint_service.go

```diff
@@ -242,27 +278,6 @@ func waitForVpcEndpointServiceDeletion(conn *ec2.EC2, serviceID string) error {
 	return err
 }
 
-func vpcEndpointServiceAttributes(d *schema.ResourceData, svcCfg *ec2.ServiceConfiguration, conn *ec2.EC2) error {
```

Comment:
Replaced by direct setting of attributes:
* https://github.com/terraform-providers/terraform-provider-aws/pull/8396/files#r277149473